CONTRIBUTING.md

```diff
@@ -1,52 +1,160 @@
+- **Test Request** – request a new test not currently available (the “test” label will be automatically applied).
+- **Security Vulnerability** – report a suspected vulnerability which risks the security of our code (the “vulnerability” label will be automatically applied, and two gliff.ai team members will be automatically assigned). Read the [gliff.ai Security Policy](https://github.com/gliff-ai/.github/blob/main/SECURITY.md) to learn more!
+
+Please remember to first determine which repository the issue should be raised in, then [search the gliff.ai repositories](https://github.com/search?q=user%3Agliff-ai+is%3Aissue+is%3Aopen) to ensure the issue you want to open is not already open! 
+
+## Create a Pull Request
+
+[{{back to navigation}}](#table-of-contents)
+
+The **Pull Request** template should be filled out as much as possible. Your pull request should include a summary of the changes you have made and your motivation behind them. Please list any dependency changes, added tests, documentation changes, migrations and complete our checklist to ensure you have completed all the necessary steps.
+
+Please remember to first determine which repository the pull request should be created in, then [search the gliff.ai repositories](https://github.com/search?q=user%3Agliff-ai+is%3Apr+is%3Aopen) to ensure the pull request you want to open is not already open! 
+
+After you submit your pull request, please assign 3 or less reviewers following our [assigning reviewers best practices](#assigning-reviewers-best-pratices) and assign any [labels](#labels) you think are applicable. Please also verify that all status checks are passing. If for some reason they are failing and you believe it’s unrelated to your change or unsure why, please leave a comment on the pull request explaining this and someone on the gliff.ai team will further investigate.
 
 ## Linting, Formatting and Testing
 
-We are using flake8 and black for linting and auto-formatting our python code and eslint and prettier.io for linting and autoformatting our TypeScript.
-Configuration files can be found in our repos.
+[{{back to navigation}}](#table-of-contents)
 
-We are using pytest for our Python tests and jest with react-testing-library for our TypeScript testing.
-Configuration files can be found in our repos.
+Our teams uses flake8 and black for linting and auto-formatting our Python code alongside pytest for our Python tests, and then eslint and prettier.io for linting and auto-formatting our TypeScript code alongside jest with react-testing-library for our TypeScript testing. Further specifics and configuration files can be found in each of our repositories.
 
-We use Pre-commit for running lints, tests and other checks locally on commit and push.
-To use pre-commit on your own machine just run:
+We use pre-commit for running lints, tests and other checks locally on commit and push. To use pre-commit on your own machine, you should run the following in your terminal:
```

Comment:
We haven't used pre-commit for about 7 months lol. It's all done in GitHub Actions now.